process.rb

```diff
@@ -280,6 +299,12 @@ def slugify(word)
       totals[:most_expensive_races] = totals[:race_totals].sort_by {|v| -v[:amount]}[0..2]
       totals.delete(:race_totals)
 
+      # Get the top 3 small contribution proprtions
+      unless totals[:small_proportion].nil?
+        totals[:largest_small_proprtion] = totals[:small_proportion].sort_by {|v| -v[:proportion]}[0..2]
```

Comment:
typo, you're missing an "o" in: `largest_small_proportion`

Reply:
fixed